GenderPayGap.UnitTests/GenderPayGap.WebUI.Tests/DatabaseTests/Return/ReturnTests.IsLateSubmission.cs

```diff
@@ -110,5 +114,36 @@ public void Is_False_When_ModifiedDate_Is_Late_And_OutOfScope(SectorTypes sector
             }
         }
 
+        [TestCase(SectorTypes.Public, ScopeStatuses.InScope)]
+        [TestCase(SectorTypes.Public, ScopeStatuses.PresumedInScope)]
+        [TestCase(SectorTypes.Private, ScopeStatuses.InScope)]
+        [TestCase(SectorTypes.Private, ScopeStatuses.PresumedInScope)]
+        [TestCase(SectorTypes.Public, ScopeStatuses.OutOfScope)]
+        [TestCase(SectorTypes.Public, ScopeStatuses.PresumedOutOfScope)]
+        [TestCase(SectorTypes.Private, ScopeStatuses.OutOfScope)]
+        [TestCase(SectorTypes.Private, ScopeStatuses.PresumedOutOfScope)]
+        public void IsFalse_When_ModifiedDate_Is_Late_And_ExcludeFromLateFlagEnforcement_Year(SectorTypes sector, ScopeStatuses scopeStatus)
+        {
+            foreach (int testYear in Global.ReportingStartYearsToExcludeFromLateFlagEnforcement)

```

Comment:
When the test fails, does it give enough information about which iteration of the foreach loop failed? Would it be better to implement this as a TestCase, like you did for the SectorType and ScopeStatus? Just so that we don't test two things in a single test.

Reply:
Not sure if possible (I'll have a look). But the array contains a single value anyways